src/components/shared/leftColumn.tsx

```diff
@@ -5,7 +5,7 @@ import { css } from '@emotion/core';
 import { remSpace } from '@guardian/src-foundations';
 import { from } from '@guardian/src-foundations/mq';
 import type { FC } from 'react';
-import React from 'react';
+import * as React from 'react';
```

Comment:
As above, are you sure this is needed?

Reply:
Just tested and makes no difference when omitted, not sure how that got in there